pycbc/workflow/pegasus_workflow.py

```diff
@@ -843,10 +851,10 @@ def insert_into_dax(self, rep_cat, sites):
     @classmethod
     def from_path(cls, path):
         """Takes a path and returns a File object with the path as the PFN."""
-        logging.warn("The from_path method in pegasus_workflow is deprecated. "
-                     "Please use File.from_path (for output files) in core.py "
-                     "or resolve_url_to_file in core.py (for input files) "
-                     "instead.")
+        DeprecationWarning("The from_path method in pegasus_workflow is "
+                           "deprecated. Please use File.from_path (for "
+                           "output files) in core.py or resolve_url_to_file "
+                           "in core.py (for input files) instead.")
```

Comment:
This will not do anything actually; if you want the warning to be printed, you need something like
```python
import warnings

warnings.warn('Blah', DeprecationWarning)
```